TransactionMobile/TransactionMobile/Presenters/LoginPresenter.cs

```diff
@@ -162,10 +185,13 @@ private async Task GetConfiguration()
                 // Do the initial logon transaction
                 await this.PerformLogonTransaction();
 
+                // Get the merchants current balance
+                await this.GetMerchantBalance();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add '.ConfigureAwait(false)' to this call to allow execution to continue in any thread.](https://app.codacy.com/manual/stuart_ferguson1/MobileApplication/pullRequest?prid=5700460)